libs/vkd3d/resource.c

```diff
@@ -3511,6 +3535,10 @@ void d3d12_desc_copy(struct d3d12_desc *dst, struct d3d12_desc *src,
         struct vkd3d_bound_buffer_range *dst_buffer_ranges = dst->heap->buffer_ranges.host_ptr;
         dst_buffer_ranges[dst->heap_offset] = src_buffer_ranges[src->heap_offset];
     }
+
+#ifdef VKD3D_ENABLE_DESCRIPTOR_QA
+    vkd3d_descriptor_debug_copy_descriptor(dst->heap, dst->heap_offset, src->heap, src->heap_offset, src->metadata.cookie);
+#endif
```

Comment:
This part may need some rebasing since `d3d12_desc_copy` is a different function now.

Reply:
Fixed.